index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,600;1,300&family=Roboto:wght@500;700&display=swap"
```

Comment:
You have added both `Open Sans` and `Roboto` here. Please, remove `Roboto`. The easy way is to create the link from google fonts again. The final url should have `Open+Sans` as `family`.